client/cli/src/commands/runcmd.rs

```diff
@@ -416,9 +416,7 @@ impl RunCmd {
 		if self.no_telemetry {
 			config.telemetry_endpoints = None;
 		} else if !self.telemetry_endpoints.is_empty() {
-			config.telemetry_endpoints = Some(
-				TelemetryEndpoints::new(self.telemetry_endpoints.clone())
-			);
+			config.telemetry_endpoints = TelemetryEndpoints::new(self.telemetry_endpoints.clone()).ok();
```

Comment:
This should return an error and not just silently put a `None`, otherwise you can pass an invalid URL, it will not work, and you have no idea why